org.jjazzlab.JJazzLab.yml

```diff
@@ -0,0 +1,97 @@
+id: org.jjazzlab.JJazzLab
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - 'org.freedesktop.Sdk.Extension.openjdk17'
+
+command: jjazzlab
+separate-locales: false
+finish-args:
+  # X11
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  # audio: pulseaudio, JACK over PipeWire
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  # printing
+  - --socket=cups
+  # files
+  - --filesystem=~/JJazzLab
+  # setup Java Runtime Environment
+  - --env=PATH=/usr/bin:/app/jre/bin:/app/bin
+  - --env=JAVA_HOME=/app/jre
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+
+  - name: soundfont
+    buildsystem: simple
+    build-commands:
+      - mkdir -p $FLATPAK_DEST/share/sounds/sf2
+      - cp JJazzLab-SoundFont.sf2 $FLATPAK_DEST/share/sounds/sf2/
+    sources:
+      - type: file
+        url: https://archive.org/download/jjazz-lab-sound-font/JJazzLab-SoundFont.sf2
+        sha256: 50ab9eaa711dd3af02544420f75881be833a876f9051c60d80c231f6387d8d80
+        # we slightly abuse the checker by using the modification time (integer) as version
+        x-checker-data:
+          type: json
+          url: https://archive.org/metadata/jjazz-lab-sound-font/files
+          version-query: '.result[] | select(.name=="JJazzLab-SoundFont.sf2") | .mtime'
+          url-query: '"https://archive.org/download/jjazz-lab-sound-font/JJazzLab-SoundFont.sf2"'
+
+  - fluidsynth2.yml
+
+  - xrdb.yml # needed to get GTK / Java theming right
```

Comment:
Add here

Reply:
I don't I fully understand what you mean here - do you mean adding xrdb to shared-modules first, and then using it here?
(note that it doesn't belong in the openjdk17 extension, since it is not a Java-specific but Netbeans Application Platform specific behaviour)

Reply:
No, add the contents of that manifest to the main one. Maintaining multiple small manifests gets hard.